src/openms/source/ANALYSIS/OPENSWATH/MRMFeatureScheduler.cpp

```diff
@@ -33,14 +33,47 @@
 // --------------------------------------------------------------------------
 
 #include <OpenMS/ANALYSIS/OPENSWATH/MRMFeatureScheduler.h>
+#include <OpenMS/ANALYSIS/OPENSWATH/MRMFeatureSelector.h>
+#include <OpenMS/KERNEL/FeatureMap.h>
 
 namespace OpenMS
 {
   MRMFeatureScheduler::MRMFeatureScheduler() {}
 
   MRMFeatureScheduler::~MRMFeatureScheduler() {}
 
-  // TODO I guess we are supposed to use setParameters()
-  void MRMFeatureScheduler::schedule_MRMFeatures_qmip() {}
-  void MRMFeatureScheduler::schedule_MRMFeatures_score() {}
+  FeatureMap MRMFeatureScheduler::schedule_MRMFeatures(MRMFeatureSelector& feature_selector, FeatureMap& features) {
+    int nn_thresholds[] = {2, 4, 6, 4};
+    bool locality_weights[] = {false, true, true, true};
+    bool select_transition_groups[] = {true, true, true, true};
+    int segment_window_lengths[] = {12, 24, 48, -1};
+    int segment_step_lengths[] = {2, 6, 12, -1};
+    bool select_highest_counts[] = {false, false, false, false};
+    String variable_types[] = {"continous", "continous", "continous", "continous"};
```

Comment:
continuous*

Could this typo be the source (or part of it) of the issues you were facing?

It's easy to mistype strings, so when you see that you're using some value in multiple places or repeatedly, just define some variable for it and then refer to its value through the identifier.

Example:
```
const String s_continuous = "continuous";
const std::vector<String> variable_types = {s_continuous, s_continuous, s_continuous, s_continuous};
// or another way to initialize it is:
const std::vector<String> variable_types(4, s_continuous);
```

Also (I should find some confirmation for this, but it's true at least for C), the syntax `type identifier[]` is equivalent to `type *identifier`. Is this what you desire? Why?